### PR TITLE
minor changes in sdk automl image object detection batch

### PR DESCRIPTION
**Pull request by ashakella**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under `# Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Notebooks` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Content` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

### CODE REVIEW

**Comment by kweinmeister** [comment]
The notebook failed with the error:
```
FailedPrecondition: 400 The train budget milli node hours must be in the range [20000, 900000] for model type CLOUD_HIGH_ACCURACY_1.
```

The current value is 1. The units are milli node hours, so 1 hour should be specified as 1000. Since the minimum for this model type is 20000 (20 hours), we should set it to 20000 for now.

**Comment by kweinmeister** [comment]
Build failure:
```
Input In [26], in <cell line: 11>()
      8 data_2 = {"content": test_item_2, "mimeType": "image/jpeg"}
     10 # Upload the test-data to Cloud storage bucket
---> 11 bucket = storage.Client(project=PROJECT_ID).bucket(BUCKET_NAME.replace("gs://", ""))
     12 blob = bucket.blob(blob_name=test_filename)
     13 data = json.dumps(data_1) + "\n" + json.dumps(data_2) + "\n"

NameError: name 'storage' is not defined
```

**Comment by kweinmeister** [comment]
Build failure:
```
----> 1 if os.getenv["IS_TESTING"]:
      2     get_ipython().system(' pip3 install --upgrade tensorflow $USER_FLAG')
      
TypeError: 'function' object is not subscriptable      
```

`os.getenv` should use parentheses rather than brackets.

File: notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb [hunk 1]
```diff
@@ -0,0 +1,1051 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:1]: <p>Change Open in Google Cloud Notebooks to Workbench AI Notebooks. Also add image.</p><p> </p><p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:1]: <p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:1]: <p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:1]: <p>change to:</p><p>BUCKET_NAME = "[your-bucket-name]" &nbsp;# @param {type:"string"}</p><p>BUCKET_URI = f"gs://{BUCKET_NAME}"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:1]: <p>BUCKET_NAME -&gt; BUCKET_URI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:1]: <p>BUCKET_NAME -&gt; BUCKET_URI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (andrewferlitsch) [inline notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:1]: <p>BUCKET_NAME -&gt; BUCKET_URI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/370/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Comment (kweinmeister) [reply notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:1]: FYI @ashakella the updated header can be found here: https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb

Comment (ashakella) [reply notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:1]: I updated the header and changed 'Vertex SDK' to 'Vertex AI SDK'

Comment (ashakella) [reply notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:1]: Changed 'Vertex SDK' to 'Vertex AI SDK'

Comment (ashakella) [reply notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:1]: Changed 'Vertex SDK' to 'Vertex AI SDK'

Comment (ashakella) [reply notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:1]: Created separate variables for bucket name and URI.

Comment (ashakella) [reply notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:1]: Replaced BUCKET_NAME with BUCKET_URI.

Comment (ashakella) [reply notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:1]: Replaced BUCKET_NAME with BUCKET_URI.

Comment (ashakella) [reply notebooks/official/automl/sdk_automl_image_object_detection_batch.ipynb:1]: Replaced BUCKET_NAME with BUCKET_URI.

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**